packages/keyring-snap-bridge/src/migrations/v1.ts

```diff
@@ -40,10 +44,11 @@ export function getScopesForAccountV1(accountV1: KeyringAccountV1): string[] {
     }
     case EthAccountType.Erc4337: {
       // EVM Erc4337 account
-      // NOTE: A Smart Contract account might not be compatible with every chain, but we still use
-      // "generic" scope for now. Also, there's no official Snap as of today that uses this account type. So
+      // NOTE: A Smart Contract account might not be compatible with every chain, in this case we just default
+      // to mainnet since we cannot really "guess" it from here.
+      // Also, there's no official Snap as of today that uses this account type. So
       // this case should never happen.
-      return [EthScopes.Namespace];
+      return [EthScopes.Mainnet];
```

Comment:
Since this migration will be applied only to the accounts created by the example Snap, maybe we should migrate them to testnet instead.